transport_nantes/press/models.py

```diff
@@ -1,3 +1,4 @@
+from email.policy import default
```

Comment:
This import has probably been made automaticaly by your IDE

Reply:
Fixed